src/ua/artcode/udiary/model/Dairy.java

```diff
@@ -8,16 +9,41 @@
     //  TODO add fields and properties (in future)
     private String id;
 
+    private User owner;
+
+    private String title;
+
     private List<Record> recordList;
 
     // Constructors:
     //  TODO add constructors (in future)
-    public Dairy(String id) {
-        this.id = id;
+
+
+    public Dairy() {
```

Comment:
There are a lot of constructors. If we need flexibility of creation we should use Builder Pattern

Reply:
Created. I've pushed it with its tests. It's trial - if your review on it is positive, I'll create such Builders for other models.